srfi-178.html

```diff
@@ -104,15 +105,15 @@ <h3>Notation</h3>
 
       <dt><i>vec</i></dt>
       <dd>
-        A heterogeneous vector; that is, it must
+        An heterogeneous vector; that is, it must
         satisfy the predicate <code>vector?</code>.
         <br/>
         <br/>
       </dd>
 
       <dt><i>bvec, to, from</i></dt>
       <dd>
-        A bitvector, i.e. it must
+        A bitvector, i.e., it must

```

Comment:
A comma is not needed here.

Reply:
That one too.  I used @abbren Texinfo instead to avoid the issue of having a double space added after a period.